scripts/kconfig_verify.py

```
@@ -21,9 +21,40 @@ def verify_kconfig(kconfig_file):
         print("No warnings found.")
 
 
+def check_kconfig_spaces(file_path):
+    """Check for space-based indentation in a Kconfig file"""
+    try:
+        # Read file content to check for spaces
+        with open(file_path, "r", encoding="utf-8") as f:
+            lines = f.readlines()
+
+        space_indent_lines = []
+        for line_num, line in enumerate(lines, 1):
+            # Check for leading spaces (skip empty lines)
+            stripped_line = line.lstrip()
+            if stripped_line and len(line) > len(stripped_line):
+                # Extract the indentation part
+                indent = line[: -len(stripped_line)] if stripped_line else line
```

`stripped_line` guaranteed truthy here.

```suggestion
            if stripped_line and len(line) > len(stripped_line):
                # Extract the indentation part
                indent = line[: -len(stripped_line)]
```